src/styles/blocks/contacts.scss

```diff
@@ -0,0 +1,33 @@
+.contacts {
```

Comment:
https://prnt.sc/S7Cr6pr1t-Lj => links it this block should change color with hover (checklist)